neurons/validator.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, config=None):
         self.synthetic_image_generator = SyntheticImageGenerator(
             prompt_type='annotation', use_random_diffuser=True, diffuser_name=None)
 
-        self._fake_prob = config.get('fake_prob', 0.5)
+        self._fake_prob = self.config.get('fake_prob', 0.5)
```

Comment:
what is this?

Reply:
When I created our CI pipeline, I allowed the probability of the validator generating a fake/real image to be set as a param. This facilitates us being able to test both flows of execution (sampling real image vs generating synthetic). 

This change is a fix for a bug that my initial unit tests didn't catch. Currently the validator tests in `tests/` test the validator forward function and reward functions, but they don't use the real `Validator.__init__` function, because that requires GPUs. This is something that we'll be able to handle when we add the CD part of the CI/CD pipeline. I'll add this to the backlog in Monday.com now!